pkg/apis/config/features.go

```diff
@@ -77,6 +77,7 @@ const (
 	FeaturePodSpecShareProcessNamespace     = "kubernetes.podspec-shareprocessnamespace"
 	FeaturePodSpecTolerations               = "kubernetes.podspec-tolerations"
 	FeaturePodSpecTopologySpreadConstraints = "kubernetes.podspec-topologyspreadconstraints"
+	FeaturePodSpecVolumesImage              = "kubernetes.podspec-volumes-image"
```

Comment:
@Fedosin I small nit, there's should be at least a enable/disable feature flag test added into `features_test.go`.

Reply:
Thanks! I added the test there.